GP/kernel_modules/random_walk.py

```diff
@@ -0,0 +1,102 @@
+# Author: Henry Moss & Ryan-Rhys Griffiths
+"""
+Molecule kernels for Gaussian Process Regression implemented in GPflow.
+"""
+
+import gpflow
+import numpy as np
+import tensorflow as tf
+
+from .kernel_utils import normalize
+
+
+class RandomWalk(gpflow.kernels.Kernel):
```

Comment:
Should we have some documentation for the class?

Reply:
Definitely. I suppose I hoped for the main logic to be looked at first but honestly I should have made a separate feature branch and done multiple PRs onto that (one for the logic, one for documentation etc)